doc/src/manual/constructors.md

```diff
@@ -539,11 +549,11 @@ that specific type parameters cannot be requested manually.
 For example, say we define a type that stores a vector along with an accurate representation of
 its sum:
 
-```julia
-type SummedArray{T<:Number,S<:Number}
-    data::Vector{T}
-    sum::S
-end
+```jldoctest
```

Comment:
that example is the wrong way around, the sum type should be wider than the vector elements